firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -280,3 +304,156 @@ uint8_t TCMDEXEC_available_telecommands(const char *args_str, TCMD_TelecommandCh
 
     return 0;
 }
+
+uint8_t TCMDEXEC_set_int_config_variable(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
+                                         char *response_output_buf, uint16_t response_output_buf_len)
+{
+    const int args_str_len = strlen(args_str);
+    char config_var_name[50];
```

Comment:
Please set 50 as a variable in `configuration.h`:  `static const uint8_t CONFIG_MAX_VARIABLE_NAME_LENGTH = 50;`